autosklearn/ensembles/ensemble_selection.py

```diff
@@ -243,7 +281,10 @@ def __str__(self):
                           enumerate(self.identifiers_)
                           if self.weights_[idx] > 0]))
 
-    def get_models_with_weights(self, models):
+    def get_models_with_weights(
+        self,
+        models: BasePipeline
+    ) -> List[Tuple[List, BasePipeline]]:
```

Comment:
Shouldn't this be `List[Tuple[float, BasePipeline]]`?

Reply:
definitely, will change this in next push.